main.go

```diff
@@ -72,63 +74,84 @@ func main() {
 			Usage:  "fail deployment if it has been superseded by another deployment. WARNING: there are some bugs in kubernetes.",
 			EnvVar: "FAIL_SUPERSEDED,PLUGIN_FAIL_SUPERSEDED",
 		},
+		cli.StringFlag{
+			Name:   "certificate-authority",
+			Usage:  "the path to a file containing the CA for kubernetes API `PATH`",
```

Comment:
I think it makes more sense to support `--certificate-authority-data` and/or `--certificate-authority`. For example, when used in Drone, we could just have an org secret `${CERTIFICATE_AUTHORITY_DATA_DEV}` with base64 encoded CA cert which is then passed onto kubectl as a file.